specification/metadata/schema/schemaid.md

```diff
@@ -2,7 +2,7 @@
 
 The Schema ID provides the reference item to associate which Schema was used for the generation of a FOCUS Dataset.
 
-The SchemaId MUST be present in the metadata. The SchemaId MUST be of String. It is RECOMMENDED for SchemaId to be a Universally Unique Identifier (UUID) or [SemVer](https://semver.org) version.
+The SchemaId MUST be present in the metadata. The SchemaId MUST be of String. It is RECOMMENDED for SchemaId to be a Universally Unique Identifier (UUID).
```

Comment:
Is `UUID` well enough understood, or does it deserve a glossary entry?

Reply:
replaced with GUID which we use in multiple places in the spec....though we don't have a spec space that defines what it is.

Reply:
I am going to resolve this. If we want to tackle adding in a guid reference, we should let future us add that.